src/extensions/wallet.ts

```diff
@@ -0,0 +1,51 @@
+import { Network, RawConfig } from '@buildwithsygma/sygma-sdk-core'
+import { Wallet } from 'ethers'
+import { GluegunToolbox, prompt } from 'gluegun'
+import { Keyring } from '@polkadot/api'
+import { InitializedWallets } from '../types'
+
+module.exports = (toolbox: GluegunToolbox) => {
```

Comment:
Why is this typed like this?
You don't seem to use toolbody so might as well just write functions that don't use like normal functions.

Reply:
That's the general convention in gluegun - bind an extension to toolbox instance and access it via that in a command